Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionAPIScripting.mm

```diff
@@ -860,6 +860,95 @@
     [manager run];
 }
 
+TEST(WKWebExtensionAPIScripting, MainWorld)
```

Comment:
i think it's make more sense to be under `WKWebExtension` since these aren't related to the scripting API

Reply:
I had similar conflicts. But `WKWebExtension` tests are more for the ObjC API, and don't test things end-to-end.